portal-backend/depmap/compound/models.py

```diff
@@ -239,7 +240,6 @@ class CompoundDose(Entity):
     )
 
     dose = Column(Float, nullable=False)
-    unit = "μM"
```

Comment:
This doesn't seem like it was used for anything before so removing it to avoid confusion